packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -256,6 +256,17 @@ function useTransition(
   return [callback => {}, false];
 }
 
+const noOp = () => {};
+
+function useEvent(event: any): any {
+  hookLog.push({primitive: 'Event', stackError: new Error(), value: event});
+  return {
+    clear: noOp,
+    listen: noOp,
+    unlisten: noOp,
+  };
```

Comment:
`useEvent` in `ReactFiberHooks` and `ReactPartialRendererHooks` returns an object with `clear` and `setListener` attributes. This one has `clear`, `listen`, and `unlisten`. Why the difference?

Reply:
That's a mistake, my bad!

Reply:
I would have expected Flow to catch this. It's supposed to ensure that all of our hooks implementations are consistent. Why is the return type of this hook `any`?